### ISSUE TITLE
Plotly scraper not working with sphinx-gallery

### ISSUE THREAD

**Issue by astroDimitrios** [plotly/plotly.py#4722]
This has been reported on a [sphinx-gallery issue](https://github.com/sphinx-gallery/sphinx-gallery/issues/1238) but I don't think it has here.

The plotly scraper is not generating thumbnails for sphinx-gallery to use. The [sphinx-gallery plotly example](https://sphinx-gallery.github.io/stable/auto_plotly_examples/plot_0_plotly.html) specifies a custom png image to use as the thumbnail so it's not working there either.

**Comment by gvwilson** [comment]
@LiamConnors would this one be yours?

**Comment by henselman-petrusek** [comment]
just checking in on this -- has there been any progress? I ran into the same issue as the original issue report on sphinx-gallery

**Comment by gvwilson** [comment]
@henselman-petrusek my apologies but we haven't been able to put any effort against this yet

**Comment by sr-murthy** [comment]
@LiamConnors Is there an update on this?

**Comment by camdecoster** [comment]
It looks like someone created a PR to address this issue. Once that's ready for review, we'll take a look.

**Comment by sr-murthy** [comment]
> It looks like someone created a PR to address this issue. Once that's ready for review, we'll take a look.

Yes I've been testing out precisely that PR, and have posted there. :)
